kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/develop/WatchMojo.java

```diff
@@ -114,7 +97,14 @@ public void executeInternal() throws MojoExecutionException {
         } catch (Exception ex) {
             throw new MojoExecutionException("An error has occurred while while trying to watch the resources", ex);
         }
+    }
 
+    protected File getManifest(KubernetesClient kubernetesClient) {
+        if (OpenshiftHelper.isOpenShift(kubernetesClient)) {
+            log.warn("OpenShift KubernetesClient detected, using Kubernetes YAML manifests");
+            log.warn("Switch to openshift-maven-client in case there are any problems");
```

Comment:
This warning looks confusing :confused:

Reply:
Any suggestion to make thinks clearer for the user?
What's basically happening is that a user is performing a k8s:watch against a Kubernetes Cluster.

Is "OpenShift cluster detected while using kubernetes-maven-plugin", "Switch to..."  clearer?

Reply:
Yeah :+1: 
```
OpenShift cluster detected, using Kubernetes manifests.
Switch to openshift-maven-plugin in case there are any problems.
```